geode-core/src/main/java/org/apache/geode/management/internal/cli/functions/ImportDataFunction.java

```diff
@@ -40,9 +40,13 @@ public void execute(FunctionContext context) {
     final Object[] args = (Object[]) context.getArguments();
     final String regionName = (String) args[0];
     final String importFileName = (String) args[1];
-    boolean invokeCallbacks = false;
+    boolean parallel = false;
     if (args.length > 2) {
```

Comment:
Don't we need to check for the total args size? which is always 4. What if an old gfsh client connected to newer server executes this function (which is not supported...yet one could do it)...in that case, won't invoke callbacks value get set to parallel?